Sources/Styleguide/DesignSystems/Typography.swift

```diff
@@ -30,6 +30,9 @@ extension Class {
     public static let fontSizeInherit = CssSelector.class("font-size-inherit")
     public static let textDecorationNone = CssSelector.class("text-decoration-none")
 
+    public static let breakWord = CssSelector.class("break-word")
+    public static let nowrap = CssSelector.class("nowrap")
```

Comment:
some new atomic styles. realized we should apply `nowrap` to our buttons so that they dont break when in constrained areas